Common/xout/xoutbase.h

```diff
@@ -44,7 +44,7 @@ using namespace std;
  */
 
 template <class charT, class traits = char_traits<charT>>
-class ITK_TEMPLATE_EXPORT xoutbase
+class xoutbase
```

Comment:
FYI Changes to the declaration of `xout` classes are likely to cause git conflicts now when trying to merge to `develop`, as I recently made them non-templated. With commit https://github.com/SuperElastix/elastix/commit/d0af9767512f7625dffc520322d421a9392c2527 (28 January 2021)